src/main/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/model/out/OcrValidationResponse.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.hmcts.reform.bulkscanccdeventhandler.model.out;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+public class OcrValidationResponse {
+
+    @JsonProperty("warnings")
+    public final List<String> warnings;
+
+    @JsonProperty("errors")
+    public final List<String> errors;
+
+    @JsonProperty("status")
+    public final String status;
+
+    @JsonCreator
+    public OcrValidationResponse(
+        @JsonProperty("warnings") List<String> warnings,
+        @JsonProperty("errors") List<String> errors,
+        @JsonProperty("status") String status
+    ) {
```

Comment:
i think response constructor doesn't need json annotations does it?

Reply:
No, it doesn't. will remove now